rideshare/Extensions/ShadowView.swift

```diff
@@ -0,0 +1,23 @@
+//
```

Comment:
This file should be named `UIView+addShadow.swift` to follow the convention we have for our other extensions!